notification-service/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "auctions-notification-service",
+  "version": "1.0.0",
+  "description": "An automated blockchain subscription service.",
+  "module": "dist/notification-service.esm.js",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "dev": "tsdx watch",
```

Comment:
It sadly only live complies. Therefore it is quite unnecessary in my opinion. 

I suggest the following:
Replace `tsdx watch` with `npm run build && node .`. This would rebuild the service and start it. I found that using this felt a lot more useful, as I did not need to run two commands. 

In addition to this: When developing the `bot`, I often found myself having to restart the entire bot during development anyway, as hot reloading works poorly with bots or these kinds of services.

Reply:
Well, according to the docs, you can provide `--onSuccess` command to run node after the compilation: https://github.com/jaredpalmer/tsdx#tsdx-watch